test/test_search.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+PARENT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, PARENT)
```

Comment:
Why are you modifying the path?  Also I'm happy to incorporate more tests, but there's already test.py - why don't you add them there?